bigchaindb/core.py

```diff
@@ -42,11 +43,13 @@ def __init__(self, bigchaindb=None):
         self.validators = None
         self.new_height = None
 
-    def init_chain(self, validators):
+    def init_chain(self, genesis):
         """Initialize chain with block of height 0"""
 
+        validator_set = [decode_validator(v) for v in genesis.validators]
         block = Block(app_hash='', height=0, transactions=[])
         self.bigchaindb.store_block(block._asdict())
+        self.bigchaindb.store_validator_set(1, validator_set)
```

Comment:
we just store the validator in init? What about later?

Reply:
We store the validator set at a given height. In the above case since the chain is being initialized so we store the `validator_set` at height `1`.